README.md

```diff
@@ -37,6 +37,10 @@ libpcap should be installed on Mac OS X by default.
 
 **Note:** A timeout of zero may cause ```pcap::Capture::next``` to hang and never return (because it waits for the timeout to expire before returning). This can be fixed by using a non-zero timeout (as the libpcap manual recommends) and calling ```pcap::Capture::next``` in a loop.
 
+## Specifying library folder
+
```

Comment:
> I've been super busy lately and I'm not sure that I have time to review PRs right now.

> However, I'll be happy to give you some control over this repository so that PRs don't sit around. Is that okay?

@ebfull That'd be great, at least for the maintenance-like fixes and PRs. E.g., I was planning to integrate clippy and lint the code, maybe clean it up a little in general, that's pure technical task. I'd probably still open a PR and wait on some comments for anything discussion-worthy. Just a thought, would it make sense to move this out to an umbrella org? (e.g. pcap-rust/pcap, idk)

Also in the plans / half-implemented already - `pcap_*fopen*()` support for both reading/writing through raw fds on unix platforms (this is important e.g. if you want to read from / write to zipped pcaps; needs some careful testing though).